sodapy/__init__.py

```diff
@@ -84,22 +84,23 @@ def create(self, name, **kwargs):
             category : must exist in /admin/metadata
             tags : array of tag strings
```

Comment:
Not from this commit, but can you change the verbiage here to the more pythonic-sounding "list of tag strings"?